ui/networker.js

```diff
@@ -0,0 +1,34 @@
+postMessage({
+    msg: "ready"
+});
+
+var workerID = "";
+
+addEventListener('message', async (e) => {
+	const msg = e.data.msg;
+	const payload = e.data.payload;
+
+	if (msg == "apiCall") {
+		const id = e.data.id;
+        let url = 'http://localhost:3333/api'
```

Comment:
this could possibly be '/api' just in case we allow the server to run on different ports...